config.py

```diff
@@ -1,5 +1,10 @@
 
 import os
 
-chromedriver_path = os.path.dirname(os.path.realpath(__file__)) + '/resources/chromedriver.exe'
+windows_chromedriver = 'chromedriver.exe'
 
+chromedriver_path = os.path.dirname(os.path.realpath(__file__)) + '/resources/' + windows_chromedriver
+
+
+homepage_url = 'https://www.coinbase.com/'
```

Comment:
name these `coinbase_homepage_url` and `coinbase_signin_page_url` because when we add more websites we will need to differentiate.